src/Okta.Sdk.IntegrationTests/UserTypesClientShould.cs

```diff
@@ -0,0 +1,269 @@
+            finally
+            {
+                await DeleteAllUserTypes();
+            }
+        }
+
+        [Fact]
+        public async Task RetrieveUserTypeById()
+        {
+            var testClient = TestClient.Create();
+
+            var testDescription = $"{SdkPrefix}:{nameof(RetrieveUserTypeById)} Test Description";
+            var testDisplayName = $"{SdkPrefix}:{nameof(RetrieveUserTypeById)} Test DisplayName";
+            var testName = $"{SdkPrefix}{nameof(RetrieveUserTypeById)}_TestUserType";
+
+            var createdUserType = await testClient.UserTypes.CreateUserTypeAsync(new UserType
+            {
+                Description = testDescription,
+                DisplayName = testDisplayName,
+                Name = testName,
+            });
+
+            try
+            {
+                createdUserType.Id.Should().NotBeNullOrEmpty();
+
+                var retrievedUserType = await testClient.UserTypes.GetUserTypeAsync(createdUserType.Id);
+                retrievedUserType.Should().NotBeNull();
+                retrievedUserType.Id.Should().Be(createdUserType.Id);
+                retrievedUserType.Description.Should().Be(testDescription);
+                retrievedUserType.DisplayName.Should().Be(testDisplayName);
+            }
+            finally
+            {
+                await DeleteAllUserTypes();
+            }
+        }
+
+        [Fact]
+        public async Task UpdateUserType()
+        {
+            var testClient = TestClient.Create();
+
+            var testDescription = $"{SdkPrefix}:{nameof(UpdateUserType)} Test Description";
+            var testDisplayName = $"{SdkPrefix}:{nameof(UpdateUserType)} Test DisplayName";
+            var testName = $"{SdkPrefix}_{nameof(UpdateUserType)}_TestUserType";
+
+            var createdUserType = await testClient.UserTypes.CreateUserTypeAsync(new UserType
+            {
+                Description = testDescription,
+                DisplayName = testDisplayName,
+                Name = testName,
+            });
+
+            try
+            {
+                createdUserType.Id.Should().NotBeNullOrEmpty();
+
+                var updatedDescription = $"{createdUserType.Description} Updated";
+                var updatedDisplayName = $"{createdUserType.Description} Updated";
+
+                var updatesToUserType = new UserType
+                {
+                    Description = updatedDescription,
+                    DisplayName = updatedDisplayName,
+                };
+
+                var updatedUserType = await testClient.UserTypes.UpdateUserTypeAsync(updatesToUserType, createdUserType.Id);
+                updatedUserType.Id.Should().Be(createdUserType.Id);
+                updatedUserType.Description.Should().Be(updatedDescription);
+                updatedUserType.DisplayName.Should().Be(updatedDisplayName);
+
+                var retrievedForValidation = await testClient.UserTypes.GetUserTypeAsync(createdUserType.Id);
```

Comment:
nit: I suggest using more descriptive names. I would rename this to `retrievedUserTypeForValidation`, so we know it is a user type.